lib/jekyll/plugin_manager.rb

```diff
@@ -70,10 +65,9 @@ def whitelist
     # Returns nothing.
     def require_plugin_files
       unless site.safe
-        plugins_path.each do |plugins|
-          Dir[File.join(plugins, "**", "*.rb")].sort.each do |f|
-            require f
-          end
+        plugins_path.each do |plugin_search_path|
+          plugin_files = Dir[File.join(plugin_search_path, "**", "*.rb")]
```

Comment:
@RochesterinNYC Yes, the `File.join` returns `"**/*.rb"` on Linux and Mac, but will return `"**\*.rb"` on Windows. :)